src/java.base/macosx/native/libjli/java_md_macosx.m

```diff
@@ -312,12 +312,12 @@ static void MacOSXStartup(int argc, char *argv[]) {
     pthread_t main_thr;
     rc = pthread_create(&main_thr, NULL, &apple_main, &args);
     if (rc != 0) {
-        JLI_ReportErrorMessageSys("Could not create main thread, return code: %s\n", rc);
+        JLI_ReportErrorMessageSys("Could not create main thread, return code:%d\n", rc);
```

Comment:
Shouldn't there be a space in between, `return code: %d`?

Reply:
I can live with both, not sure what is preferred.  In the corresponding fprintf output of 8339475 there is no space.